Mathlib/FieldTheory/RatFunc/Basic.lean

```diff
@@ -525,6 +527,21 @@ theorem ofFractionRing_algebraMap (x : K[X]) :
     ofFractionRing (algebraMap _ (FractionRing K[X]) x) = algebraMap _ _ x := by
   rw [← mk_one, mk_one']
 
+def toFractionRingRAlgEquiv {R : Type*} [CommSemiring R]
+  [IsDomain K] [Algebra R K]:
```

Comment:
```suggestion
def toFractionRingAlgEquiv (R : Type*) [CommSemiring R] [Algebra R K]:
```

We usually just write `algEquiv` without saying the `R` in front. Also, I think there is already an `IsDomain` instance above. Also, I changed `R` to explicit because it cannot be inferred from the arguments.